Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -531,6 +531,44 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+  public override func visit(_ node: CanImportExprSyntax) -> SyntaxVisitorContinueKind {
+    if shouldSkip(node) {
+      return .skipChildren
+    }
+
+    if let label = node.label,
+      label.presence == .missing
+    {
+      addDiagnostic(
+        label,
+        .canImportWrongSecondParameterLabel,
+        handledNodes: [label.id]
+      )
+
+      [node.unexpectedBetweenLabelAndColon?.id, node.colon?.id, node.versionTuple?.id].compactMap { $0 }.forEach { handledNodes.append($0) }
```

Comment:
```suggestion
      handledNodes.append(contentsOf: [node.unexpectedBetweenLabelAndColon?.id, node.colon?.id, node.versionTuple?.id].compactMap { $0 })
```

reads nice IMO because it shows the action that is being performed